.goreleaser.yaml

```diff
@@ -64,6 +64,25 @@ builds:
     ldflags:
       - -s -w -X main.Version={{.Version}} -X main.Commit={{.ShortCommit}} -X main.BuildDate={{.Date}}
 
+  - id: linux_aarch64
+    main: ./cli/main.go
+    binary: rill
+    goos:
+      - linux
+    goarch:
+      - arm64
```

Comment:
Looks like go env didn't provide literal arch name as `aarch`, so I'd assume `arm64` might be fit in this case?

Reply:
Yeah, `aarch64` refers to the same architecture as `arm64`